crates/driver/src/boundary/settlement.rs

```diff
@@ -409,6 +409,15 @@ fn to_token_amount(asset: &eth::Asset) -> TokenAmount {
     }
 }
 
+fn to_big_decimal(value: bigdecimal::BigDecimal) -> num::BigRational {
+    let (x, exp) = value.into_bigint_and_exponent();
```

Comment:
This feels like a function that could have a unit test because it's easy to switch up numerator and denominator or have an off by one error.